src/pyskindose/settings_pyskindose.py

```diff
@@ -71,12 +71,24 @@ def __init__(self, settings: Union[str, dict]):
             tmp = settings
 
         self.mode = tmp[KEY_PARAM_MODE]
+        self.k_tab_val = tmp[KEY_PARAM_K_TAB_VAL]
         self.rdsr_filename = tmp[KEY_PARAM_RDSR_FILENAME]
         self.estimate_k_tab = tmp[KEY_PARAM_ESTIMATE_K_TAB]
-        self.k_tab_val = tmp[KEY_PARAM_K_TAB_VAL]
+
         self.phantom = PhantomSettings(ptm_dim=tmp['phantom'])
         self.plot = Plotsettings(plt_dict=tmp['plot'])
 
+    def print_parameters(self):
```

Comment:
Could you add a unit test to test this print functionality?

Reply:
Sure, added in 6dcbf4c7e495c6d1a5bc9ea9201498b92dd5ba21